cluster/examples/kubernetes/ceph/cluster.yaml

```diff
@@ -77,6 +77,12 @@ spec:
   # enable the crash collector for ceph daemon crash collection
   crashCollector:
     disable: false
+  cleanupPolicy:
+    # cleanupPolicy should only be added to the cluster when the cluster is about to be deleted.
+    # After any field of the cleanup policy is set, Rook will stop configuring the cluster as if the cluster is about
+    # to be destroyed in order to prevent these settings from being deployed unintentionally.
+    deleteDataDirOnHosts: ""
```

Comment:
How about mentioning in the comment exactly what the value needs to be set to? Then they don't have to search the docs.